src/Components/ImportHAR.jsx

```diff
@@ -1,29 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDropzone } from 'react-dropzone';
+import { Button } from 'react-bootstrap';
+import { useNetwork } from '../state/network/provider';
```

Comment:
i think the next pr with linting change will catch this, but `{ useNetwork }` should be grouped with styles

also i prefer `./` to start at the current directory like you are with `Styles` below

Reply:
I will fix `./` in next PR, need to disable eslint rule `import/no-useless-path-segments` for that.